src/plugins/navigation/public/top_nav_menu/_index.scss

```diff
@@ -1,15 +1,15 @@
 .kbnTopNavMenu__wrapper {
   z-index: 5;
+}
 
-  .kbnTopNavMenu {
-    padding: $euiSizeS 0;
+.kbnTopNavMenu {
+  padding: $euiSizeS 0;
 
-    .kbnTopNavItemEmphasized {
-      padding: 0 $euiSizeS;
-    }
+  .kbnTopNavItemEmphasized {
+    padding: 0 $euiSizeS;
   }
+}
```

Comment:
The menu can now be rendered outside the `TopNavMenu` component due to the portal. I had to move the `.kbnTopNavMenu` styles outside of `.kbnTopNavMenu__wrapper`.

Note: there will probably be some style tweaking depending on the display, but this will have to be handled during the next step in https://github.com/elastic/kibana/pull/72331

Reply:
The next EUI update will likely address some of this. I can assist Michail with any additional touch ups. Thanks!